src/compute-client/src/controller/instance.rs

```diff
@@ -1076,8 +1082,18 @@ where
             );
         }
 
-        self.compute
-            .send(ComputeCommand::CreateDataflow(augmented_dataflow));
+        // Skip the actual dataflow creation for a materialized view with an empty `as_of`.
+        // (Happens e.g. for the bootstrapping of a REFRESH AT mat view that is past its last
+        // refresh.)
+        if as_of.is_empty() && augmented_dataflow.is_materialized_view() {
```

Comment:
Above the comment states that we don't create a dataflow when `as_of.is_empty()`, but here there is an additional constraint about a materialized view, so that's inconsistent. Do we need the constraint on the materialized view? If not I'd prefer to just drop it.

Reply:
Deleting the MV check now, as agreed on zoom.